kolibri/plugins/learn/assets/src/views/classes/ClassAssignmentsPage.vue

```diff
@@ -57,15 +56,15 @@
       LearnAppBarPage,
     },
     mixins: [commonCoreStrings, commonLearnStrings],
-    setup(_, { root }) {
+    setup(props) {
       const { fetchClass, getClass, getClassActiveLessons, getClassActiveQuizzes } =
         useLearnerResources();
 
-      const classId = root.$router.currentRoute.params.classId;
-      const classroom = computed(() => getClass(classId));
+      const classId = computed(() => props.classId);
+      const classroom = computed(() => getClass(classId.value));
       const className = computed(() => (get(classroom) ? get(classroom).name : ''));
-      const activeLessons = computed(() => getClassActiveLessons(get(classId)));
-      const activeQuizzes = computed(() => getClassActiveQuizzes(get(classId)));
+      const activeLessons = computed(() => getClassActiveLessons(get(classId.value)));
```

Comment:
Shouldn't have to call `get` and use the `.value` accessor - only one or the other is needed.

Reply:
Fixed! thank you